sale_import_rest/controllers/main.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+from odoo import _, http
+from odoo.exceptions import ValidationError
+
+
+class SaleImportBaseController(http.Controller):
+    @http.route("/sale_order/post", type="json", auth="api_key", methods=["POST"])
```

Comment:
It will be better to depend on base rest and add a RestController.
Json will be supported in the right way and the documentation of the API will be exposed